scripts/sync-and-test-bucket.sh

```diff
@@ -37,7 +37,7 @@ destination_bucket_uri="s3://${destination_bucket}"
 # Translate Hugo redirects into a file we'll use for making 301 redirects later. Note that
 # we do this before syncing the site, as having the redirects file available for reference
 # can be helpful for debugging purposes.
-node scripts/translate-redirects.js "$build_dir" "$(pulumi -C infrastructure config get redirectDomain || echo '')"
+node scripts/translate-redirects.js "$build_dir" "$(pulumi -C infrastructure config get redirectDomain 2>/dev/null || echo '')"
```

Comment:
To keep things a little simpler, could we put `"$(pulumi -C infrastructure config get redirectDomain 2>/dev/null || echo '')"` on a separate line? That would make things a bit more readable, IMHO.